### PR TITLE
Doc update

### PR DESCRIPTION
**Pull request by wziww**
### Summary

Add `TEST_NGINX_BINARY`  environment variables description for dev-manual doc

### CODE REVIEW

File: doc/dev-manual-cn.md [hunk 1]
```diff
@@ -99,3 +99,4 @@ Makefile rules:
 2. 然后通过 cpanm 来安装 test-gninx：`sudo cpanm --notest Test::Nginx IPC::Run > build.log 2>&1 || (cat build.log && exit 1)`
 3. 然后 clone 最新的源码：`git clone https://github.com/openresty/test-nginx.git`
 4. 通过 perl 的 `prove` 命令来加载 test-nginx 的库，并运行 `/t` 目录下的测试案例集:`prove -Itest-nginx/lib -r t`
+>可通过 TEST_NGINX_BINARY 环境变量来设置执行的 nginx 路径 , 如 TEST_NGINX_BINARY=/usr/local/bin/openresty prove -Itest-nginx/lib -r t
```
Comment (juzhiyuan) [inline doc/dev-manual-cn.md]: Please remove the whitespace between `路径` & `，`

File: doc/dev-manual.md [hunk 2]
```diff
@@ -99,4 +99,5 @@ Makefile rules:
 2. Then install `test-gninx` via `cpanm`:：`sudo cpanm --notest Test::Nginx IPC::Run > build.log 2>&1 || (cat build.log && exit 1)`
 3. clone source code：`git clone https://github.com/openresty/test-nginx.git`;
 4. Load the `test-nginx` library with perl's `prove` command and run the test cases in the `/t` directory: `prove -Itest-nginx/lib -r t`
+>The environment variables TEST_NGINX_BINARY can set the execution path of nginx , such as TEST_NGINX_BINARY=/usr/local/bin/openresty prove -Itest-nginx/lib -r t
```
Comment (juzhiyuan) [inline doc/dev-manual.md]: `The environment variables` --> `The environment variable`

**Comment by juzhiyuan** [comment]
Thanks for your contribution!

File: doc/dev-manual.md (hunk 2 above)
Comment (wziww) [reply doc/dev-manual.md]: https://github.com/apache/incubator-apisix/pull/755/commits/eea2d0dde30c1c9c1bd821e0102d3fb847d5fa30

File: doc/dev-manual-cn.md (hunk 1 above)
Comment (wziww) [reply doc/dev-manual-cn.md]: https://github.com/apache/incubator-apisix/pull/755/commits/2ef4e0d428b222f261bcf5bfad3dac4f642eb26f

File: doc/dev-manual-cn.md [hunk 3]
```diff
@@ -99,3 +99,4 @@ Makefile rules:
 2. 然后通过 cpanm 来安装 test-gninx：`sudo cpanm --notest Test::Nginx IPC::Run > build.log 2>&1 || (cat build.log && exit 1)`
 3. 然后 clone 最新的源码：`git clone https://github.com/openresty/test-nginx.git`
 4. 通过 perl 的 `prove` 命令来加载 test-nginx 的库，并运行 `/t` 目录下的测试案例集:`prove -Itest-nginx/lib -r t`
+>可通过 TEST_NGINX_BINARY 环境变量来设置执行的 nginx 路径,如 TEST_NGINX_BINARY=/usr/local/bin/openresty prove -Itest-nginx/lib -r t
```
Comment (juzhiyuan) [inline doc/dev-manual-cn.md]: image

The comma character should be in tradition Chinese version.

Comment (wziww) [reply doc/dev-manual-cn.md]: Also fix the semicolon character. https://github.com/apache/incubator-apisix/pull/755/commits/8550ed24d6992bac84524fd9353319878d242a53

File: doc/dev-manual-cn.md [hunk 4]
```diff
@@ -98,4 +98,5 @@ Makefile rules:
 1. 先安装 perl 的包管理器 cpanminus
 2. 然后通过 cpanm 来安装 test-gninx：`sudo cpanm --notest Test::Nginx IPC::Run > build.log 2>&1 || (cat build.log && exit 1)`
 3. 然后 clone 最新的源码：`git clone https://github.com/openresty/test-nginx.git`
-4. 通过 perl 的 `prove` 命令来加载 test-nginx 的库，并运行 `/t` 目录下的测试案例集:`prove -Itest-nginx/lib -r t`
+4. 通过 perl 的 `prove` 命令来加载 test-nginx 的库，并运行 `/t` 目录下的测试案例集：`prove -Itest-nginx/lib -r t`
+>可通过 TEST_NGINX_BINARY 环境变量来设置执行的 nginx 路径，如 TEST_NGINX_BINARY=/usr/local/bin/openresty prove -Itest-nginx/lib -r t
```
Comment (membphis) [inline doc/dev-manual-cn.md]: how about this style? I think it is more clear.

```
... ...
3. xxxx
4. ****, 并运行 `/t` 目录下的测试案例集:
    * 直接运行：`prove -Itest-nginx/lib -r t`
    * 指定 Ningx 二进制路径：`TEST_NGINX_BINARY=/usr/local/bin/openresty prove -Itest-nginx/lib -r t`
```

Comment (wziww) [reply doc/dev-manual-cn.md]: nice suggestion,if we have other recommendations,we can also list them in this way.

File: doc/dev-manual-cn.md [hunk 5]
```diff
@@ -98,4 +98,6 @@ Makefile rules:
 1. 先安装 perl 的包管理器 cpanminus
 2. 然后通过 cpanm 来安装 test-gninx：`sudo cpanm --notest Test::Nginx IPC::Run > build.log 2>&1 || (cat build.log && exit 1)`
 3. 然后 clone 最新的源码：`git clone https://github.com/openresty/test-nginx.git`
-4. 通过 perl 的 `prove` 命令来加载 test-nginx 的库，并运行 `/t` 目录下的测试案例集:`prove -Itest-nginx/lib -r t`
+4. 通过 perl 的 `prove` 命令来加载 test-nginx 的库，并运行 `/t` 目录下的测试案例集：`prove -Itest-nginx/lib -r t`
```
Comment (membphis) [inline doc/dev-manual-cn.md]: We can remove  `prove -Itest-nginx/lib -r t`, it is duplicated.

**Pull request merged**